src/js/outreach/views/opt-in_views.js

```diff
@@ -0,0 +1,153 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import 'scss/modules/buttons.scss';
+import 'scss/modules/forms.scss';
+
+import './opt-in.scss';
+
+const OptInView = View.extend({
+  template: hbs`
+    <div class="opt-in__icon opt-in__icon--warn">{{fat "hand-wave"}}</div>
+    <h2 class="opt-in__heading-text">Hi, we need to confirm your contact info. Please enter your information below, so that we can share health resources with you.</h2>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your first name</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-first-name"
+        placeholder="Enter your first name"
+        value="{{ firstName }}"
+      />
+    </div>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your last name</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-last-name"
+        placeholder="Enter your last name"
+        value="{{ lastName }}"
+      />
+    </div>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your date of birth</label>
+      <input
+        type="text"
```

Comment:
I think we should make this the date input just like the dob on the outreach login.

Reply:
I just updated it to use `type="date"`.